### PR TITLE
Feature/cross tx

### PR DESCRIPTION
**Pull request by pengin7384**
Done
- Add cross_test_job (worker)
- Add cross_shard_test (test)
- Add shard_test (test, moved from cli)
- Update cli

### CODE REVIEW

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Review by liayoo — APPROVED** [review]
LGTM with a suggestion. Thanks! 👍

File: test_type/cross_shard.js [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+  dataPool.sort((a, b) => {
+    return a.sentAt < b.sentAt ? -1 : a.sentAt > b.sentAt ? 1 : 0;
+  });
+
+  // Slice by time zone
+  const roundList = [];
+  for (const test of testList) {
+    // TODO: Fix matchedList === [] error
+    console.log(`test.jobList[0].output: ${JSON.stringify(test.jobList[0].output, null, 2)}`);
+    const startTime = test.jobList[0].output.matchedList[0].sentAt;
```
[the remark this answers is not in the document]
Comment (pengin7384) [reply test_type/cross_shard.js]: Good idea! Done

**Comment by pengin7384** [comment]
PTAL @liayoo @platfowner

File: test_type/cross_shard.js [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+    }
+  }
+  dataPool.sort((a, b) => {
+    return a.sentAt < b.sentAt ? -1 : a.sentAt > b.sentAt ? 1 : 0;
+  });
+
+  // Slice by time zone
+  const roundList = [];
+  for (const test of testList) {
+    const startTime = _.get('test.jobList[0].output.matchedList[0].sentAt', 0);
```
Comment (liayoo) [inline test_type/cross_shard.js:156]: Should this be `_.get(test, 'jobList[0].output.matchedList[0].sentAt', 0)` ?

Comment (pengin7384) [reply test_type/cross_shard.js]: Right!!! Thanks for catching my mistake. Done

Comment (liayoo) [reply test_type/cross_shard.js]: Thanks for the quick updates! LGTM!

**Comment by pengin7384** [comment]
Merged thanks!

**Pull request merged**